src/card/styled-components.js

```diff
@@ -13,7 +13,7 @@ export const Action = styled<{}>('div', ({$theme: {typography}}) => ({
   ...typography.font450,
 }));
 
-export const Body = styled<{}>('div', ({$theme}) => {
+export const Body = styled<{}>('p', ({$theme}) => {
```

Comment:
I am a bit worried, that with this if someone wants to add multiple paragraphs for the card, we'll end up with the same issues as with the modal. I've just sent an email to the SEO consultant to ask for recommendation